Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDeviceConfigurationEndpointProtectionPolicyWindows10/MSFT_IntuneDeviceConfigurationEndpointProtectionPolicyWindows10.schema.mof

```diff
@@ -0,0 +1,361 @@
+    [Write, Description("Configures the firewall to prevent merging Firewall Rule policies from group policy with those from local store instead of ignoring the local store rules. When PolicyRulesFromGroupPolicyNotMerged and PolicyRulesFromGroupPolicyMerged are both true, PolicyRulesFromGroupPolicyMerged takes priority.")] Boolean PolicyRulesFromGroupPolicyNotMerged;
+    [Write, Description("Configures the firewall to allow the host computer to respond to unsolicited network traffic of that traffic is secured by IPSec even when stealthModeBlocked is set to true. When SecuredPacketExemptionBlocked and SecuredPacketExemptionAllowed are both true, SecuredPacketExemptionAllowed takes priority.")] Boolean SecuredPacketExemptionAllowed;
+    [Write, Description("Configures the firewall to block the host computer to respond to unsolicited network traffic of that traffic is secured by IPSec even when stealthModeBlocked is set to true. When SecuredPacketExemptionBlocked and SecuredPacketExemptionAllowed are both true, SecuredPacketExemptionAllowed takes priority.")] Boolean SecuredPacketExemptionBlocked;
+    [Write, Description("Prevent the server from operating in stealth mode. When StealthModeRequired and StealthModeBlocked are both true, StealthModeBlocked takes priority.")] Boolean StealthModeBlocked;
+    [Write, Description("Allow the server to operate in stealth mode. When StealthModeRequired and StealthModeBlocked are both true, StealthModeBlocked takes priority.")] Boolean StealthModeRequired;
+    [Write, Description("Configures the firewall to block unicast responses to multicast broadcast traffic. When UnicastResponsesToMulticastBroadcastsRequired and UnicastResponsesToMulticastBroadcastsBlocked are both true, UnicastResponsesToMulticastBroadcastsBlocked takes priority.")] Boolean UnicastResponsesToMulticastBroadcastsBlocked;
+    [Write, Description("Configures the firewall to allow unicast responses to multicast broadcast traffic. When UnicastResponsesToMulticastBroadcastsRequired and UnicastResponsesToMulticastBroadcastsBlocked are both true, UnicastResponsesToMulticastBroadcastsBlocked takes priority.")] Boolean UnicastResponsesToMulticastBroadcastsRequired;
+};
+[ClassVersion("1.0.0")]
+class MSFT_MicrosoftGraphWindowsFirewallRule
+{
+    [Write, Description("The action the rule enforces. If not specified, the default is Allowed. Possible values are: notConfigured, blocked, allowed."), ValueMap{"notConfigured","blocked","allowed"}, Values{"notConfigured","blocked","allowed"}] String Action;
+    [Write, Description("The description of the rule.")] String Description;
+    [Write, Description("The display name of the rule. Does not need to be unique.")] String DisplayName;
```

Comment:
Please make sure you mark this property as Required instead of Write. See other comment.

Reply:
fixed

Reply:
I still see the property marked as Write here. It should be
`[Required, Description("The display name of the rule. Does not need to be unique.")] String DisplayName;`